code/modules/mob/living/emote.dm

```diff
@@ -212,6 +257,29 @@
 			else
 				playsound(H, pick('sound/voice/human/manlaugh1.ogg', 'sound/voice/human/manlaugh2.ogg'), 50, 1)
 
+/datum/emote/living/oof
+	key = "oof"
+	key_third_person = "oofs"
+	message = "oofs."
+	message_mime = "oofs silently!"
+	emote_type = EMOTE_AUDIBLE
+
+/datum/emote/living/oof/can_run_emote(mob/living/user, status_check = TRUE)
+	. = ..()
+	if(. && iscarbon(user))
+		var/mob/living/carbon/C = user
+		return !C.silent
+
+/datum/emote/living/oof/run_emote(mob/user, params)
+	. = ..()
+	if(. && ishuman(user))
+		var/mob/living/carbon/human/H = user
+		if(H.dna.species.id == "human" && (!H.mind || !H.mind.miming))

```

Comment:
needs a 1% chance to play a different oof

Reply:
Like what?

Reply:
https://www.youtube.com/watch?v=f49ELvryhao